en/micro-integrator/mkdocs.yml

```diff
@@ -379,7 +379,8 @@ nav:
         - 'Message Relay' : 'setup/message_builders_formatters/message-relay.md'
         - Observability:
           - 'Observability Deployment Strategy': 'setup/observability/observability-deployment-strategy.md'
-          - 'Setting up Cloud Native Observability Deployment': 'setup/observability/setting-up-minimum-basic-observability-deployment.md'
+          - 'Setting up Cloud Native Observability in a VM': 'setup/observability/setting-up-minimum-basic-observability-deployment.md'
```

Comment:
I am thinking this should be:
- Setting up Cloud-Native Observability for a VM Deployment
- Setting up Cloud-Native Observability for a K8s Deployment

Because the subject is the MI deployment (VM or K8s)
wdyt?

Reply:
Done